### ISSUE TITLE
Enhancement - better feedback from mapbox when things are not the way they should be

### ISSUE THREAD

**Issue by AnneEstoppey** [mapbox/mapboxgl-jupyter#133]
Hello

I am still new to mapbox, and suffer a little from lack of feedback from mapbox when things are not how they should be. The dataset just doesn't display on the map, without any indication of what could be wrong.

For instance today, I merged to dataframes into a new one. 
There were both pointsets.

I ran the geojson file and proceeded to display the map.

(Note that I had already displayed several maps with mapbox with similar datasets: no problems.)

Anyhow, my map didn't show my points.

After a lot of head scratching, I realized that the index of my new dataframe had a lot of doubles, due to the merging.

So I reset the indexation and re-ran the geojson file.

BINGO: my points appeared on my mapbox map!

It would have been really nice to have some error message telling me for instance that something was amiss with my indexation.

Thanks for your attention,
Cheers :-)
Anne
(MacOS, Jupyter Notebook, Python 3.xx)

**Comment by ryanbaumann** [comment]
Hi @AnneEstoppey, thanks for your ticket.  To make this ticket actionable, I recommend that we scope it to a specific error message to add to the Mapboxgl-Jupyter library.  Based on your description, how does this sound?

> Add error message - `Dataframe indexes must be unique for each row.  Try Dataframe.resetIndex() to fix this error`

**Comment by AnneEstoppey** [comment]
Hello, sorry for delayed answer, yes I think this error message looks good!

Have you reproduced this issue on your side?

Cheers :-)

**Comment by ryanbaumann** [comment]
We just released v0.10.0 on pypi - can you test if this issue still exists in your sample notebook after running `pip install mapboxgl --upgrade`?

**Comment by AnneEstoppey** [comment]
Yes I can try that, but I am a bit afraid to upgrade now that I have almost finished my edX assignment with mapbox. 

Is it possible to roll back to version 0.9.0 just in case?
How?

Cheers
Anne

**Comment by ryanbaumann** [comment]
No worries - I know the feeling! 
 Use `pip install --force-reinstall mapboxgl==0.9.0` in the same python environment or kernel that you use in your Jupyter Notebook to downgrade to a specific version, if you need to.
